integration-libs/punchout/core/facade/punchout.service.ts

```diff
@@ -269,6 +286,11 @@ export class PunchoutService implements PunchoutFacade {
     return this.requestPunchoutSessionCommand.execute(punchoutSessionId);
   }
 
+  submitRequisition(cancelRequisition: boolean): void {
```

Comment:
i always thought facade need to return Observable, if it works fine with void that's better.

Reply:
Yup, observable or void only, feel free to see my PR about restrict facade-descriptor: https://github.com/SAP/spartacus/pull/20253/files